public/locales/af/mining-view.json

```
@@ -0,0 +1,15 @@
+{
+    "floor": "Vloer",
+    "current-block-time": "Huidige bloktyd",
+    "your-reward-is": "Jou beloning is",
+    "connection-to-node-lost": "Mynwerker verbinding verloor. Wag asseblief vir mynwerker verbinding of begin weer.",
```

or I would put it completely different and a more direct translation to
```suggestion
    "connection-to-node-lost": "Kontak met mynwerker verloor. Wag asseblief vir mynwerker om kontak te maak of begin weer.",
```